dislash/slash_commands/slash_core.py

```diff
@@ -76,12 +75,12 @@ def __init__(self, func, *, name=None, connectors=None, **kwargs):
         if cooldown is None:
             try:
                 # Assuming that it's discord.py 1.7.0+
-                self._buckets = CooldownMapping(cooldown, BucketType.default)
-            except:
+                self._buckets = CooldownMapping(cooldown)

```

Comment:
```suggestion
                self._buckets = CooldownMapping(cooldown, BucketType.default)
```
What is the reason for dropping this?